Demo/src/main/AndroidManifest.xml

```diff
@@ -37,6 +37,7 @@
 
                 <data android:scheme="https" />
                 <data android:host="mobile-sdk-demo-site-838cead5d3ab.herokuapp.com" />
+                <data android:pathPrefix="/braintree-payments" />
```

Comment:
The `/braintree-payments` path was added to the app link URL at some point which broke the app link flow in the demo app.